### PR TITLE
Send debt reminder

### PR DESCRIPTION
**Pull request by Hopekodi**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/transaction/show.blade.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
                                         {{ \Carbon\Carbon::parse($transaction->createdAt)->diffForhumans() }}</h6>
                                 </div>
                                 <div>
-                                    <a href="#" class="btn btn-warning mr-3"> Send Debt Reminder </i>
+                                    <a href="" data-toggle="modal" data-target="#bs-example-modal-sm2" class="btn btn-warning mr-3"> Send Debt Reminder </i>
```
Comment (Douglasokolaa) [inline resources/views/backend/transaction/show.blade.php:21]: can you rename `#bs-example-modal-sm2` to `#sendReminderModal`

File: resources/views/backend/transaction/show.blade.php [hunk 2]
```diff
@@ -347,6 +347,36 @@ class="mr-2 text-uppercase">
     </div>
 </div>
 
+
+{{-- Modal for send reminder --}}
+            <div class="modal fade" id="bs-example-modal-sm2" tabindex="-1" role="dialog"
```
Comment (Douglasokolaa) [inline resources/views/backend/transaction/show.blade.php:352]: also rename `bs-example-modal-sm2` to `sendReminderModal`

File: resources/views/backend/transaction/show.blade.php [hunk 3]
```diff
@@ -347,6 +347,36 @@ class="mr-2 text-uppercase">
...
+                                @csrf
+                                <input type="hidden" name="transaction_id" value="{{old('transaction_id', $transaction->_id)}}">
+                               
+                                <div class="form-group">
+                                            {{-- <textarea class="form-control"
+                                                        id="exampleInput2" placeholder="Message"></textarea> --}}
+                                
+                                    <label>Message</label>
+                                    <p>
+                                        <span>characters remaining: <span id="rem_exampleInput2" title="140"></span></span>
```
Comment (Douglasokolaa) [inline resources/views/backend/transaction/show.blade.php:367]: can we possible to rename the id to `rem_reminderMessage`

File: resources/views/backend/transaction/show.blade.php [hunk 4]
```diff
@@ -347,6 +347,36 @@ class="mr-2 text-uppercase">
...
+                                    </p>
+                                    <textarea name="message" class="countit form-control" id="exampleInput2" placeholder="Message" maxlength="140"></textarea>
```
Comment (Douglasokolaa) [inline resources/views/backend/transaction/show.blade.php:369]: also rename the id to `reminderMessage`

**Comment by Jeremiahiro** [comment]
Please effect necessary changes

**Pull request merged**